src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -107,6 +107,11 @@ public abstract class WebRequestPSCmdlet : PSCmdlet
         /// </summary>
         internal int _maximumFollowRelLink = int.MaxValue;
 
+        /// <summary>
+        /// Maximum number of Redirects to follow, caches WebSession.MaximumRedirection.
+        /// </summary>
+        internal int _maximumRedirection;
```

Comment:
Please make the comment better.

Reply:
I have changed the comment, but not sure if it qualifies as better. Please use more actionable feedback or my changes may be misguided and make things worse, not better.